src/config/load-dotenv.ts

```diff
@@ -0,0 +1,12 @@
+import * as dotenv from 'dotenv';
```

Comment:
I'm not sure we need this file? for environment variables we can use `.env` file which shouldn't be submitted to Github (it's part of `.gitignore` file)

Reply:
This file is for using the `.env` file - it uses the `dotenv` package to load the values from `.env` into `process.env`. `acbs.config.ts` and `doc.config.ts` were already both doing this, I just:

- centralised the logic into 1 file
- added an override that if `process.env.NODE_ENV === 'test'` then we should actually use the environment variables in `.env.test` instead (which is committed) so we can use different environment variables for the api tests

Reply:
we can set up in Github different environments :)  including Dev/Test/etc. So we can say: locally load the environments from `.env` file. In Github we load them from Github Secrets (which could be different values altogether). In Azure we load them from Azure Vaults, etc. 

The only reason I'm saying it is because we would have to maintain it in different places.

Reply:
Sorry, the separate `.env.test` file is not for the environment we're deploying to, it's for locally running the api-test suite. I didn't want to use the real `.env` file in our api-test suite because the test needs to refer to the values and we don't want to commit them.

For example, for the api tests to check that the app sends the correct login name and password to ACBS when authenticating, the tests need to refer to the login name and password. We don't want this to be the real login name and password because we don't want to commit this, so instead we tell the test to expect some fake values. We then need to make the API actually use those fake values. If we do this in `.env` then the developer will have to edit `.env` every time they swap between developing and running the api tests. Instead, we can use a separate `.env.test` file and tell the API to read from that file instead of `.env` when it's running for the API tests.

Another alternative would be for the tests to read from `.env` as well (so we don't commit the values) but I think this is more prone to error

Reply:
I see! Good point. 
Let me have a think on this and revert back to it on Monday

Reply:
We discussed this on Slack, here's a summary for when we refer back to the PR:

- Abhi and Gabriel had a chat and suggested the approach [here](https://razinj.dev/how-to-mock-process-env-in-jest/), where we have `jest` override `process.env` before the tests and return it to normal after the tests.
- I was concerned because this would mean the api tests don't test that we load `.env` (or `.env.test`) correctly, which is risky because it depends a lot on import order and our linter can automatically change import order.
- Gabriel explained that only development uses a `.env` file anyway, the deployed app will set environment variables directly, so it's not a problem that we don't test loading from the `.env` file.

So I'll swap the API tests to override `process.env` directly